proto/workflow.proto

```diff
@@ -131,9 +131,24 @@ message ExecuteWorkflowRequest {
   // Ex. "e782592faf56da05cc0a243220689135e807958f"
   string commit_sha = 4;
 
+  reserved 5;
+
+  // URL of the pushed repo.
+  // Ex. "https://github.com/some-user/acme"
+  string pushed_repo_url = 6;
+
   // Branch corresponding to the SHA to be checked out.
   // Ex. "cool-feature"
-  string branch = 5;
+  string pushed_branch = 7;
+
+  // URL of the target repo.
+  // Ex. "https://github.com/acme-inc/acme"
+  string target_repo_url = 8;
+
```

Comment:
nit: comment on pushed_repo* and target_repo* what these are:
 pushed repo is the repository this commit was pushed to
 target repo is the repository this commit will be merged into.

or something similar?